docs/user-guide/device-metrics.md

```diff
@@ -50,4 +50,128 @@ spec:
     system:
       allowList: 
           name: system-allow-list
+```
+## Metrics receiver
+### Overview
+The devices can be configured to write the metrics to a remote server. The client in the device uses [Prometheus Remote Write API](https://docs.google.com/document/d/1LPhVRSFkGNSuU1fBd81ulhsCPR4hkSZyyBj1SZ8fWOM/edit#heading=h.p12mxouu8g0h) (see also [Prometheus Integrations](https://prometheus.io/docs/operating/integrations/)). The device writes metrics until all are written. It then waits 5 minutes for more metrics to be collected.
```

Comment:
> The device writes metrics until all are written

I'd like to suggest to reword/clarify a bit. e.g. "all are written": what does it means "all"?

Reply:
I changed it. please recheck